src/rsmq/cmd/delete_message.py

```diff
@@ -31,7 +33,9 @@ def get_transaction(self):
         queue_base = self.queue_base
         queue_key = self.queue_key
         message_id = self.get_id
-
+        # decode to string when provided as bytes
+        if isinstance(message_id, bytes):
+            message_id = message_id.decode('utf-8')
```

Comment:
to avoid issues with `tx.zrem()`  we may want to adjust the message id specifically for generated rc/fr keys - but it seems to work fine as-is